changelog.d/20230822_160326_corentin.garcia_update_readme.md

```diff
@@ -0,0 +1,5 @@
+### Changed
```

Comment:
I don't think this needs to be in the changelog: the target audience for release notes are people who already have the app installed, they don't really care about how to do it :)